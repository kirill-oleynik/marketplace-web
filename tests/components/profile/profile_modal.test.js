const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const ProfileModal = require('../../../app/components/profile/profile_modal').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <ProfileModal
        t={() => {}}
        isOpen={true}
        closeModal={() => {}}
        currentUser={{}}
      />
    );

    expect(toJSON(component)).toMatchSnapshot();
  });
});
