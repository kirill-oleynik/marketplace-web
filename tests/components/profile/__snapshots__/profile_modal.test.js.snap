// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<Modal
  autoFocus={true}
  backdrop={true}
  backdropTransitionTimeout={150}
  className="main-modal"
  fade={true}
  isOpen={true}
  keyboard={true}
  modalTransitionTimeout={300}
  zIndex={1050}
>
  <Container
    tag="div"
  >
    <div
      className="text-right pt-30"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        <i
          className="icon icon-cross font-16 in-black-035"
        />
      </button>
    </div>
    <Row
      className="main-modal__content"
      tag="div"
    >
      <Col
        md={
          Object {
            "offset": 3,
            "size": 6,
          }
        }
        sm={
          Object {
            "offset": 2,
            "size": 8,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
        xs="12"
      >
        <p
          className="font-24 font-700 mb-30"
        />
        <div
          className="main-tabs"
        >
          <Nav
            className="main-tabs__nav"
            tabs={true}
            tag="ul"
          >
            <NavItem
              tag="li"
            >
              <NavLink
                className="active"
                onClick={[Function]}
                tag="a"
              />
            </NavItem>
            <NavItem
              tag="li"
            >
              <NavLink
                className=""
                onClick={[Function]}
                tag="a"
              />
            </NavItem>
          </Nav>
          <TabContent
            activeTab="profile"
            tag="div"
          >
            <TabPane
              tabId="profile"
              tag="div"
            >
              <Connect(Translate(ProfileFormContainer)) />
            </TabPane>
            <TabPane
              tabId="password"
              tag="div"
            >
              <Connect(Translate(PasswordFormContainer)) />
            </TabPane>
          </TabContent>
        </div>
      </Col>
    </Row>
  </Container>
</Modal>
`;
